exercises/calc/calc.py

```diff
@@ -12,30 +12,36 @@
     srp1
 )
 
-
+# Define custom packet class for P4calc
 class P4calc(Packet):
     name = "P4calc"
-    fields_desc = [ StrFixedLenField("P", "P", length=1),
-                    StrFixedLenField("Four", "4", length=1),
-                    XByteField("version", 0x01),
-                    StrFixedLenField("op", "+", length=1),
-                    IntField("operand_a", 0),
-                    IntField("operand_b", 0),
-                    IntField("result", 0xDEADBABE)]
-
+    # Define fields for the P4calc packet
+    fields_desc = [ StrFixedLenField("P", "P", length=1), 
+                    StrFixedLenField("Four", "4", length=1),  
+                    XByteField("version", 0x01),  
+                    StrFixedLenField("op", "+", length=1),  
+                    IntField("operand_a", 0),  
+                    IntField("operand_b", 0),  
+                    IntField("result", 0xDEADBABE)]  
```

Comment:
Are there any actual changes in the lines that are the list assigned to `fields_desc`?  If they are whitespace only changes, can they be eliminated?

Reply:
There are no whitespaces. Just a comment added. This change popped up due to my document editing style.

Reply:
'git diff' does not show lines as changed unless _something_ about it has changed.  If nothing has changed visually, usually that means that whitespace has changed, e.g. the original line had tabs, and the modified lines have spaces (or vice versa).

I would prefer if we not make such kinds of changes unless there is a good reason, as such lines make the diffs more confusing.